src/main/java/ch/uzh/ifi/seal/soprafs20/entity/Game.java

```diff
@@ -156,11 +158,11 @@ public Long getActivePlayerId() {
         return activePlayerId;
     }
 
-    public List<Integer> getScore() {
+    public Map<Long, Integer> getScore() {
```

Comment:
In order to provide the statistics of each user after the game ends, I changed it to Map